tests/ci/docker_images_check.py

```diff
@@ -16,36 +16,52 @@
 from clickhouse_helper import ClickHouseHelper, prepare_tests_results_for_clickhouse
 from stopwatch import Stopwatch
 
+from typing import List, Tuple
+
 NAME = "Push to Dockerhub (actions)"
 
-def get_changed_docker_images(pr_info, repo_path, image_file_path):
+
+def get_changed_docker_images(
+    pr_info: PRInfo, repo_path: str, image_file_path: str
+) -> Tuple[List[Tuple[str, str]], str]:
     images_dict = {}
     path_to_images_file = os.path.join(repo_path, image_file_path)
     if os.path.exists(path_to_images_file):
-        with open(path_to_images_file, 'r') as dict_file:
+        with open(path_to_images_file, "r") as dict_file:
             images_dict = json.load(dict_file)
     else:
-        logging.info("Image file %s doesnt exists in repo %s", image_file_path, repo_path)
+        logging.info(
+            "Image file %s doesnt exists in repo %s", image_file_path, repo_path
+        )
 
-    dockerhub_repo_name = 'yandex'
+    dockerhub_repo_name = "yandex"
```

Comment:
Still "yandex"?

Reply:
It's automatic [black](https://github.com/psf/black/) formatting. The PR it far from be finished, and I prefer to focus here on aarch64.

Other way, it will be quite complex to review.

But sure, makes sense to edit docker/images.json and lines 53,54 too. Especially since there's a bug, probably

Reply:
Didn't we move to "ClickHouse" dockerhub?

Reply:
It's here https://github.com/ClickHouse/ClickHouse/pull/32911/files#diff-5b4ed5d9824e0bd86ffb7baa16bb1d35417ca8bff01970350455c88179b5af07R53-R54

Reply:
It's gone to arguments parser for good now